packages/devextreme/js/__internal/ui/calendar/m_calendar.ts

```diff
@@ -756,8 +804,9 @@ class Calendar<
     $element.toggleClass(CALENDAR_RANGE_CLASS, selectionMode === 'range');
 
     this._renderBody();
-    // @ts-expect-error ts-error
-    $element.append(this.$body);
+    if (this.$body) {
+      $element.append(this.$body);
```

Comment:
Seems this.$body exists always, we render it on initMarkup, so we can mark it with '!' and remove condition
```
  $body!: dxElementWrapper;
  ```

Reply:
https://github.com/DevExpress/DevExtreme/pull/30599/commits/ec2f610d374bc1eb469a6ab73af78a85080ffa5d